src/CMakeLists.txt

```diff
@@ -22,12 +23,11 @@ message(STATUS "ZMQ found => " "${ZMQ_LIB}")
 message(STATUS "HAL found => " "${HAL_LIB}")
 message(STATUS "MATRIX LIB found => " "${MATRIX_LIB}")
 message(STATUS "MATRIX ZMQ LIB found => " "${MATRIX_ZMQ_LIB}")
+message(STATUS "MATRIX_PROTOS_LIB LIB found => " "${MATRIX_PROTOS_LIB}")
```

Comment:
Just FYI: We need to make a macro for this. Something that fails with a fatal error.